packages/common/src/restHelpers.ts

```diff
@@ -180,7 +337,7 @@ export function convertExtent(
   outSR: ISpatialReference,
   geometryServiceUrl: string,
   authentication: UserSession
-): Promise<IExtent> {
```

Comment:
What was the problem with the type, please?

Reply:
@MikeTschudi If the project call does not return an array with two things it can resolve undefiend. I just thought any was more fitting. However, I'm not sure that would ever be hit. It seems like calls that do not fail but also do not produce valid output just return "NaN" coordinate values for our points.